.github/workflows/platform-test.yml

```diff
@@ -64,6 +65,20 @@ jobs:
       - name: "Run platform matrix test"
         run: vendor/bin/phpunit --group=platform
 
+      - name: "Install Coverage driver"
+        if: matrix.update-packages == '' && (matrix.php-version == '8.2' || matrix.php-version == '8.3' || matrix.php-version == '8.4')
+        uses: "shivammathur/setup-php@v2"
+        with:
+          coverage: "pcov"
+          php-version: "${{ matrix.php-version }}"
+          ini-file: development
+          extensions: pdo, mysqli, pgsql, pdo_mysql, pdo_pgsql, pdo_sqlite, mongodb
+
+      - name: "Run infection"
+        if: matrix.update-packages == '' && (matrix.php-version == '8.2' || matrix.php-version == '8.3' || matrix.php-version == '8.4')
+        run: make infection
+        #run: vendor/bin/infection --git-diff-filter=AM
```

Comment:
I don't want to run it on "platform tests" but on the traditional tests in "build.yml" instead.

Reply:
Does this benefit from being run after `vendor/bin/phpunit` or should it be a completely separate job instead?